timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimeLockResource.java

```diff
@@ -89,17 +89,21 @@ TimeLockServices getOrCreateServices(String namespace) {
         return servicesByNamespace.computeIfAbsent(namespace, this::createNewClient);
     }
 
-    public int numberOfClients() {
+    public int getNumberOfActiveClients() {
```

Comment:
Hmm, I think this broke some of the timelock tests. Should be an easy fix though

Reply:
D'oh.  This is now fixed.